### ISSUE TITLE
[Broken Website] Can not play live games or puzzles on Chess.com - Chrome

### ISSUE THREAD

**Issue by Mister-Curious** [darkreader/darkreader#8040]
- Can the issue be reproduced in a fresh browser profile? Unsure 
  
  - Is there an existing issue about this? Unsure 
  Please fill out every section of this report, removing any that are not needed.

# Broken Website Report

## Website Address
https://www.chess.com/play/online

## Steps To Reproduce
Go to https://www.chess.com 
log into account
play live chess or load puzzle

## Problematic Observation
Screen is totally white

## Screenshots
N/A

## System Information
- OS: Windows 10 Pro
- Browser: Chrome Version 97.0.4692.99 (Official Build) (64-bit)
- Dark Reader Version: 4.9.43

## Additional Context
Issue is not present on Microsoft edge. Issue can not be resolved by disabling pluggin on page only, and must be disabled in Chrome extensions to make web page load.

**Comment by Mister-Curious** [comment]
It doesn't seem to have an issue when I log into Chrome as 'Guest', but it appears that Dark Reader is not enabled on the guest profile, so I am not sure what you can get out of that.

**Comment by Gusted** [comment]
Hi @Mister-Curious!

I'm sorry, let me rephrase it, Could you try this on a fresh chrome profile, with dark reader installed?

Regards,
Gusted

**Comment by Mister-Curious** [comment]
ok thanks for the clarification. To answer this, there are no issues with a new profile and DR installed.

**Comment by Gusted** [comment]
That most likely means that their is a interfering extension or setting on that profile. Could you check if you're able to identify such thign/

**Comment by Mister-Curious** [comment]
Thanks and sorry for not replying sooner (the problem seemed to have fixed itself up until recently). 
I just tried it again with a guest profile and it seems to be broken. All the whites are grey.
